Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -2353,4 +2354,14 @@ private IQueryable<BaseItemEntity> TranslateQuery(
 
         return baseQuery;
     }
+
+    /// <inheritdoc/>
+    public async Task<bool> ItemExistsAsync(Guid id)
+    {
+        var dbContext = await _dbProvider.CreateDbContextAsync().ConfigureAwait(false);
+        await using (dbContext.ConfigureAwait(false))
+        {
+            return dbContext.BaseItems.Any(f => f.Id == id);
```

Comment:
```suggestion
            return await dbContext.BaseItems.AnyAsync(f => f.Id == id).ConfigureAwait(false);
```

Since the function is already async